superset/assets/src/visualizations/deckgl/layers/Polygon/Polygon.jsx

```diff
@@ -1,31 +1,40 @@
-import d3 from 'd3';
+/* eslint no-underscore-dangle: ["error", { "allow": ["", "__timestamp"] }] */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
 import { PolygonLayer } from 'deck.gl';
-import { flatten } from 'lodash';
-import { colorScalerFactory } from '../../../../modules/colors';
-import { commonLayerProps } from '../common';
+
+import AnimatableDeckGLContainer from '../../AnimatableDeckGLContainer';
+import Legend from '../../../Legend';
+import { getBuckets, getBreakPointColorScaler } from '../../utils';
+
+import * as common from '../common';
```

Comment:
better import specific functions `import { fn }` than `import *` to fully benefit from tree-shaking.

Reply:
Definitely, will fix.